.github/workflows/test.yml

```diff
@@ -6,7 +6,7 @@ on:
     - 'release/*'
   pull_request:
 env:
-  CORE_REPO_SHA: 47483865854c7adae7455f8441dab7f814f4ce2a
+  CORE_REPO_SHA: 3b813eb9921e709538dd1b07fa7a5f93600fbec1
```

Comment:
The SHA needs to be updated every time in the contrib repo because it runs tests using the latest version of core